src/SortableContainer/index.js

```diff
@@ -339,12 +341,21 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 				if (transitionDuration) {
 					node.style[`${vendorPrefix}TransitionDuration`] = `${transitionDuration}ms`;
 				}
+
+				const margin = getElementMargin(node);
+				// margins merge so we subtract them
+				const delta = (
+					axis === 'x'
+					? this.width  + this.margin.left + this.margin.right  - margin.left
+					: this.height + this.margin.top  + this.margin.bottom - margin.top
+				);
+
```

Comment:
> I'm not sure it's that uncommon.

I don't know, but I personally never used it and can't think of a use case for it in a list.

> Have you looked at how other libraries handle this?

All the other reordering libraries that I found change the order of the element in the DOM, so don't need to handle this case.
One technique used to animate changes between two states of the DOM that stood-out was the [FLIP animation](https://aerotwist.com/blog/flip-your-animations/#the-general-approach) used by [react-flip-move](https://github.com/joshwcomeau/react-flip-move)

> Also, have you tried with negative margins 🤔

No, I can't say that I have. I'll try it and let know